app/services/workflow_find_service.rb

```diff
@@ -1,7 +1,12 @@
 class WorkflowFindService
   def find(tag_attrs_arr)
     tag_attrs_arr.collect do |tag_attrs|
-      TagLink.where(tag_attrs).pluck(:workflow_id).first
+      # TODO: need to retrieve tag name based on :object_id from remote app
+      #  tag_attrs.merge(:tag_name => tag_name)
+      object_id = tag_attrs[:object_id]
+      workflow_id = TagLink.where(tag_attrs.except(:object_id)).pluck(:workflow_id).first
+
+      WorkflowResponse.new(object_id, workflow_id)
```

Comment:
I don't think we should image how the caller will use the api, or based on how our UI will use it. The API should be simple and generic enough. The return should be either an array of `workflow_id` or an array of `workflow` full object.
To align with GET `/workflows` that returns the full objects, we may want to do the same here.